### PR TITLE
docs(provideRouter): Show explicit usage of LocationStrategy & HashLocation

### PR DESCRIPTION
**Pull request by jhuntoo**
I got stuck debugging my app for something so simple as not passing HashlocationStragey, so maybe this will save somebody else's time !

### CODE REVIEW

**Comment by brandonroberts** [comment]
The default strategy is the PathLocationStrategy. I'm also leaning towards this being a separate section in the docs as it may require a little more explanation. Thoughts?

cc: @MikeRyan52

**Comment by brandonroberts** [comment]
@jhuntoo you still want to take a look at adding this?

File: README.md [hunk 1]
```diff
@@ -92,16 +92,18 @@ npm install @ngrx/router @ngrx/core --save
 
   ```ts
   import { provideRouter } from '@ngrx/router';
+  import {  HashLocationStrategy, PathLocationStrategy } from '@angular/common';
 
   bootstrap(App, [
-    provideRouter(routes)
+    provideRouter(routes, PathLocationStrategy) // Or HashLocationStrategy
```
Comment (jhuntoo) [inline README.md]: Oops the example I had was wrong anyway.

**Comment by jhuntoo** [comment]
@brandonroberts I've just added a more detailed section about LocationStrategy, though I'd argue that we keep it in the main README, to avoid a potentially common pitfall for others.

File: docs/overview/location-strategy.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+## PathLocationStrategy
+
+Urls will appear like example.com/myroute
+
+```ts
+  import { provideRouter } from '@ngrx/router';
+  import {  PathLocationStrategy } from '@angular/common';
+
+  bootstrap(App, [
+    provideRouter(routes, PathLocationStrategy)
```
Comment (brandonroberts) [inline docs/overview/location-strategy.md]: You don't need to do anything but `provideRouter(routes)` here. PathLocationStrategy is provided by default

File: docs/overview/location-strategy.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+# Location Strategy
+
+Angular 2 supports two kinds of routing known as location strategies.
+- [PathLocationStrategy](https://angular.io/docs/js/latest/api/router/PathLocationStrategy-class.html)
+- [HashLocationStrategy](https://angular.io/docs/js/latest/api/router/HashLocationStrategy-class.html)
+
+
+The location strategy can be registerd with ngrx/router during bootstrap, and if not specified explicitly, it will default to PathLocationStrategy.
```
Comment (brandonroberts) [inline docs/overview/location-strategy.md]: `registered`

File: docs/overview/location-strategy.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+## HashLocationStrategy
+
+Urls will appears like example.com/#/myroute
+
+ ```ts
+  import { provideRouter } from '@ngrx/router';
+  import {  HashLocationStrategy } from '@angular/common';
```
Comment (brandonroberts) [inline docs/overview/location-strategy.md]: Only need one space around the imports on each side

**Comment by brandonroberts** [comment]
Can you fix the issues and rebase on top of master? Thanks!

**Comment by jhuntoo** [comment]
@brandonroberts fixed, rebased, squashed.

File: docs/overview/location-strategy.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+Urls will appear like example.com/myroute
+
+```ts
+  import { provideRouter } from '@ngrx/router';
+  import { PathLocationStrategy } from '@angular/common';
+
+  bootstrap(App, [
+    provideRouter(routes, PathLocationStrategy)
+  ]);
```
Comment (brandonroberts) [inline docs/overview/location-strategy.md]: Remove PathLocationStrategy since it's the default

**Comment by brandonroberts** [comment]
Thanks!

**Pull request merged**